releng/org.eclipse.n4js.releng.utils/src/org/eclipse/n4js/antlr/N4JSAntlrGeneratorFragment2.xtend

```diff
@@ -144,7 +144,7 @@ class N4JSAntlrGeneratorFragment2 extends N4AntlrGeneratorFragment2 {
 		}
 	}
 
-	private def boolean hasSyntheticTerminalRule() {
+	protected override boolean hasSyntheticTerminalRule() {
```

Comment:
Is it different from the impl in the super class?

Reply:
Yes it is. This method is now a `protected` method in the super class `XtextAntlrGeneratorFragment2`